pl_examples/hydra_examples/conf/pl/checkpoint/modelckpt.yaml

```diff
@@ -0,0 +1,12 @@
+# @package _group_
+cls: pytorch_lightning.callbacks.ModelCheckpoint
```

Comment:
planning to move the rest of the "core" config files to Structured Configs?

Reply:
Done!